docs/FAQ.md

```diff
@@ -6,4 +6,40 @@ description: Frequently Asked Questions
 
 ## What is the precision of MOB?
 
-The atomic unit for MOB is picoMOB, which is 1e-12. You need u64 to represent MOB, and many frameworks, DBs, and languages top out at u32 or i64. This is why Full-Service json responses are all strings. For i64 issues there is technically no loss of precision, but you need to cast back to u64 when fetching data.
\ No newline at end of file
+The atomic unit for MOB is picoMOB, which is 1e-12. You need u64 to represent MOB, and many frameworks, DBs, and languages top out at u32 or i64. This is why Full-Service json responses are all strings. For i64 issues there is technically no loss of precision, but you need to cast back to u64 when fetching data.
+
+
+
+## How can I make importing of accounts faster?
+When calling [`import_account`](https://mobilecoin.gitbook.io/full-service-api/api-endpoints/v2/account/account/import_account), scanning generally starts at the Origin block (#1).
+By using the **first_block_index** param, a user can specify the starting block for that entropy if it is known. Newly created accounts are given a starting block index. 
```

Comment:
Maybe add something like:

Any transactions that occurred before the **first_block_index** will not be tracked, potentially leading to invalid balances or denied transactions.